shared/dto/global/search-widget-data-params.ts

```diff
@@ -0,0 +1,11 @@
+export const SEARCH_WIDGET_DATA_OPERATORS = {
+  EQUALS: '=',
+  NOT_EQUALS: '!=',
+} as const;
+
+export const VALID_SEARCH_WIDGET_DATA_OPERATORS = Object.values(
+  SEARCH_WIDGET_DATA_OPERATORS,
+);
+
+export type WidgetDataFilterOperatorType =
+  (typeof SEARCH_WIDGET_DATA_OPERATORS)[keyof typeof SEARCH_WIDGET_DATA_OPERATORS];
```

Comment:
I'm failing to  see the advantage of this approach creating 3 elements to make use of a single const containing possible operators

Could you explain this please?

Reply:
@alexeh maybe you need to implement something in the code and use `SEARCH_WIDGET_DATA_OPERATORS` as an enum. The second one is for a validation and the third the type?

What would you do?

Reply:
can't we use just a enum? same as https://github.com/Vizzuality/4-growth/pull/115#discussion_r1780681440